src/main/java/sirius/kernel/commons/Strings.java

```diff
@@ -263,6 +268,16 @@ public static String firstFilled(String... values) {
         return null;
     }
 
+    /**
+     * Returns if the given string is an HTTP(S) URL.
+     *
+     * @param value the string to check
+     * @return <tt>true</tt> if the given string is an HTTP(S) URL, <tt>false</tt> otherwise
+     */
+    public static boolean isUrl(@Nullable String value) {
```

Comment:
Required: rename to isHttpUrl, since `file://...`, `ftp://` are also URLs (the protocol is not part of it)